CHANGELOG.md

```diff
@@ -3,6 +3,24 @@ SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
 SPDX-License-Identifier: Apache-2.0
 -->
 
+## v3.10.3 (?)
+
+### New features / functionalities
+
+### Changed defaults / behaviours
+
+### Deprecated / removed options and commands
+
+### Security Related Fixes
+
+### Bug Fixes
+
+-   manual_glidein_submit now correctly sets idtokens in the EncryptedInputFiles (PR #280)
```

Comment:
It is `(issue #283)` or `(issue #283, PR#284)` 
Then you can rebase and add it to the 3.10.2 bug fixes to include it in 3.10.2